data/brands/shop/variety_store.json

```diff
@@ -42,7 +42,876 @@
         "shop": "variety_store"
       }
     },
+    {{
+  "properties": {
```

Comment:
Sorry, but you put too much wrong information

I guess maybe you just paste the whole document in that place

Reply:
Ikr, something went wrong